OrbitLinuxTracing/TracerThread.cpp

```diff
@@ -711,8 +741,8 @@ void TracerThread::PrintStatsIfTimerElapsed() {
   if (stats_.event_count_begin_ns + EVENT_STATS_WINDOW_S * NS_PER_SECOND <
       MonotonicTimestampNs()) {
     double actual_window_s = static_cast<double>(MonotonicTimestampNs() -
-                                                 stats_.event_count_begin_ns) /
-                             NS_PER_SECOND;
+        stats_.event_count_begin_ns) /
```

Comment:
Are you using CLion's Ctrl-Alt-L? Use `clang-format`.

Reply:
Actually, CLion should use clang-format. Strange